src/backend/optimizer/path/allpaths.c

```diff
@@ -356,9 +356,16 @@ set_plain_rel_pathlist(PlannerInfo *root, RelOptInfo *rel, RangeTblEntry *rte)
 	if (root->config->enable_seqscan)
 		pathlist = lappend(pathlist, seqpath);
 
-	/* Consider index and bitmap scans */
-	create_index_paths(root, rel,
-					   &indexpathlist, &bitmappathlist);
+	/* TODO: this might have been too ambitious of a re-ordering */
```

Comment:
@t1mursadykov , could you explain why?

Reply:
There was a comment from commit which i back ported.  It may be connected to https://github.com/greenplum-db/gpdb/pull/6665#issuecomment-453920358.  

According to the PR story they figured out in which cases they don't need to generate paths for indexscan and decided to let create paths in some cases which could be wrong